src/analyze/analyze-security.c

```diff
@@ -1764,6 +1779,11 @@ static int assess(const SecurityInfo *info,
                         d = strdup("Service runs in special boot phase, option is not appropriate");
                         if (!d)
                                 return log_oom();
+                } else if (access_skip(a, policy)) {
+                        badness = UINT64_MAX;
+                        d = strdup("Option excluded by policy, skipping");
+                        if (!d)
+                                return log_oom();
```

Comment:
why a separate field for this? wouldn't a weight of zero more naturally and less redundantly encode this?